src/test/java/htsjdk/samtools/cram/structure/CramCompressionRecordTest.java

```diff
@@ -12,19 +13,24 @@
  * Created by vadim on 28/09/2015.
  */
 public class CramCompressionRecordTest extends HtsjdkTest {
-    @Test
-    public void test_getAlignmentEnd() {
+    @DataProvider(name = "deltaEncodingTrueFalse")
+    private Object[][] tf() {
+        return new Object[][] { {true}, {false}};
+    }
+
+    @Test(dataProvider = "deltaEncodingTrueFalse")
```

Comment:
Can we add tests for the various cases of `getAlignmentSpan` (and UNINITIALIZED_SPAN if appropriate) ?